pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -59,6 +60,9 @@ func (nameScoping *DefaultNameScopingImpl) Handle(command utils.CommandEnum, clu
 			newQuery = strings.Replace(r.RequestURI, r.URL.Query().Get("name"), r.URL.Query().Get("name")+r.Header.Get(headers.AuthZTenantIdHeaderName), 1)
 			config.Config.Labels[headers.OriginalNameLabel] = r.URL.Query().Get("name")
 		}
+		// Replace network reference with network ID, force Network-scoped alias for DNS use.
```

Comment:
Goes to the engine with network name without the prefix and fail. Additional name scoping is needed when --net-alias is provided.